### PR TITLE
docs: condense "Writing policies" section

### PR DESCRIPTION
**Pull request by agilgur5**
## Description

Closes #19 by heavily condensing the section in general instead of fixing outdated links, based on the discussion there

- link out to the [docs page on "Writing Policies"](https://docs.kubewarden.io/tutorials/writing-policies) instead

- misc `markdownlint` newline autofixes

## Test

See the [rendered markdown](https://github.com/kubewarden/.github/blob/8d1cd8bbd77013ad9bd71bc06954a16b9c4fb718/profile/README.md#writing-policies-)

### Tradeoff

n/a

### Potential improvement

n/a

### CODE REVIEW

**Review by kkaempf — CHANGES_REQUESTED** [review]
Thanks for your PR !

However, while I agree that the table should be condensed, the language keywords (Rust, Go, ...) better stay for visibility.

**Comment by flavio** [comment]
I've applied the changes requested by @kkaempf 

@agilgur5: thanks a lot for your contribution! :heart:

File: profile/README.md [hunk 1]
```diff
@@ -23,21 +22,13 @@ Don't forget to take a look at [`kwctl`](https://github.com/kubewarden/kwctl), o
...
-| Rego - Gatekeeper | [:octocat:](https://github.com/kubewarden/gatekeeper-policy-template) | [Rego built-ins](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions) | ✔️ | ❌ | 🔝 |
-| DotNet     |❌  | [:octocat:](https://github.com/kubewarden/policy-sdk-rust) | ✔️ | ✔️ | ↗️  |
 
+Kubewarden allows you to write policies using a variety of programming languages, including Rust, Go, Rego, CEL and others.
 
-Can't find your favorite language? 🔍 Reach out to us and let's have a chat!
+These are some useful resources to get you started:
+ - [Step-by-step tutorials](https://docs.kubewarden.io/tutorials/writing-policies) inside of our documentation 📖
+ - [Policy SDK](https://github.com/kubewarden/community/#policies-sdks): overview of the Policy SDKs available, and their maturity level 📚
+ - [Policy templates](https://github.com/kubewarden/community/#policies-templates): boilerplate code to help you get started 🚀
```
Comment (agilgur5) [inline profile/README.md:31]:
```suggestion

- [Step-by-step tutorials](https://docs.kubewarden.io/tutorials/writing-policies) inside of our documentation 📖
- [Policy SDKs](https://github.com/kubewarden/community/#policies-sdks): overview of the Policy SDKs available, and their maturity level 📚
- [Policy templates](https://github.com/kubewarden/community/#policies-templates): boilerplate code to help you get started 🚀 
```

- adjust spacing to pass `markdownlint` (new line before the list and no space before the hyphen)
- "SDK" -> "SDKs"

**Review by agilgur5 — COMMENTED** [review]
> I've applied the changes requested by @kkaempf

Thanks @flavio, I like the [content changes](https://github.com/kubewarden/.github/pull/20/commits/23924ba115c9d71d688ad425cf7035f297b4bcfe) you made!
(although the spacing doesn't quite pass `markdownlint` 😅 )

**Pull request merged**